cypress/e2e/log-in.cy.js

```diff
@@ -0,0 +1,14 @@
+/*global describe, it*/
+import HomePage from '../src/pages/HomePage'
+
+describe('Login and Logout tests', () => {
+    it('Login and Logout first user in page', () => {
+        const homePage = new HomePage()
+
+        homePage.visit().checkPageUrl().header.clickOnLogInButton()
+
+        homePage.logInModal
+            .logInWithCredentials('testUser3444', 'testUser3444')
+            .header.checkUserName('testUser3444')
```

Comment:
Find out how to avoid hardcoded values. Maybe it's better to create some kind of UserHandler and use github secrets or variables